lci_strategic_plugin/config/parameters.yaml

```diff
@@ -35,6 +35,15 @@ desired_distance_to_stop_buffer : 15.0
 # Double: The minimum period in seconds which a maneuver plan must cover if the plugin wishes to control the whole plan
 min_maneuver_planning_period : 15.1
 
+# Double: Update time interval of carma streets. Units in s
+delta_t : 1.0
+
+# Double: Minimum vehicle reaction time. Units in s
+reaction_time : 5.0
+
+#Int: Switch case for usecase 2 and usecase 3; 0 for UC2 and 1 for UC3
+case_switch : 0
```

Comment:
Can you describe the usage of this flag? Since the only values listed are 0 and 1 it implies that choosing UC2 or UC3 is mutually exclusive. This is not what we want. The vehicle should dynamically determine if the upcoming intersection supports Map/Spat only (UC2) or if it is equipped with carma-streets based on the broadcast from the intersection.

For example if the west intersection did not have carma-streets but the east intersection did, we chould be able to go through both intersections in a single run without changing the configuration. 

If you want a flag to disable UC2 or UC3 behavior that is fine, but the dynamic choice needs to be supported.